includes/admin/settings/register-settings.php

```diff
@@ -983,6 +983,14 @@ function edd_get_registered_settings() {
 					),
 				),
 				'file_downloads' => array(
+					'require_login_to_download' => array(
+						'id'            => 'require_login_to_download',
+						'name'          => __( 'Require Login', 'easy-digital-downloads' ),
+						'desc'          => __( 'Check this if you want to require a user to login before file download links deliver the file.', 'easy-digital-downloads' ),
```

Comment:
```suggestion
						'desc'          => __( 'Require a user to login before file download links deliver the file.', 'easy-digital-downloads' ),
```
I would love to see us move away from "check this" language.